packages/compass-sidebar/src/components/multiple-connections/active-connections/active-connection.tsx

```diff
@@ -0,0 +1,144 @@
+import {
+  Icon,
+  IconButton,
+  css,
+  palette,
+  spacing,
+  useHoverState,
+} from '@mongodb-js/compass-components';
+import type { ConnectionInfo } from '@mongodb-js/connection-info';
+import React, { useCallback } from 'react';
+import ServerIcon from '../icons/server-icon';
+
+const iconStyles = css({
+  flex: 'none',
+});
+
+const toggleBtnStyles = css({
+  color: palette.gray.dark2,
+  backgroundColor: 'none',
+  boxShadow: 'none',
+  '&:hover': {
+    backgroundColor: 'none',
+    boxShadow: 'none',
+  },
```

Comment:
Here I'm trying my best to suppress the background/shadow/something on LG buttons - it is not sufficient. Ideas welcome, otherwise I'll come back to this within https://jira.mongodb.org/browse/COMPASS-7773